tests/test_iqm_backend.py

```diff
@@ -137,3 +137,11 @@ def test_error_on_empty_circuit_list(backend):
             qubit_mapping={},
             shots=42
         )
+
+
+def test_client_close(backend):
```

Comment:
Same nit-pickiness as in related `cirq-on-iqm` PR: maybe rename to `test_close_client` to be in line with the corresponding method name `IQMSampler.close_client()`.